app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -100,444 +99,449 @@ export function addComposerUploadMarkdownResolver(resolver) {
 export function cleanUpComposerUploadMarkdownResolver() {
   uploadMarkdownResolvers = [];
 }
-export default Component.extend(
-  ComposerUploadUppy,
-  ComposerVideoThumbnailUppy,
-  {
-    classNameBindings: ["showToolbar:toolbar-visible", ":wmd-controls"],
-
-    editorClass: ".d-editor",
-    fileUploadElementId: "file-uploader",
-    mobileFileUploaderId: "mobile-file-upload",
-
-    composerEventPrefix: "composer",
-    uploadType: "composer",
-    uppyId: "composer-editor-uppy",
-    composerModel: alias("composer"),
-    composerModelContentKey: "reply",
-    editorInputClass: ".d-editor-input",
-    shouldBuildScrollMap: true,
-    scrollMap: null,
-    processPreview: true,
-
-    uploadMarkdownResolvers,
-    uploadPreProcessors,
-    uploadHandlers,
-
-    init() {
-      this._super(...arguments);
-      this.warnedCannotSeeMentions = [];
-      this.warnedGroupMentions = [];
-    },
-
-    @discourseComputed("composer.requiredCategoryMissing")
-    replyPlaceholder(requiredCategoryMissing) {
-      if (requiredCategoryMissing) {
-        return "composer.reply_placeholder_choose_category";
-      } else {
-        const key = authorizesOneOrMoreImageExtensions(
-          this.currentUser.staff,
-          this.siteSettings
-        )
-          ? "reply_placeholder"
-          : "reply_placeholder_no_images";
-        return `composer.${key}`;
-      }
-    },
-
-    @discourseComputed
-    showLink() {
-      return (
-        this.currentUser && this.currentUser.link_posting_access !== "none"
-      );
-    },
-
-    @observes("focusTarget")
-    setFocus() {
-      if (this.focusTarget === "editor") {
-        putCursorAtEnd(this.element.querySelector("textarea"));
-      }
-    },
-
-    @discourseComputed
-    markdownOptions() {
-      return {
-        previewing: true,
-
-        formatUsername,
+export default Component.extend(ComposerUploadUppy, {
```

Comment:
This is the only relevant change in this file, removal of ComposerVideoThumbnailUppy mixin